assets/stories/skate-components/navMenu.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import NavMenu from "../../src/components/nav/navMenu"
+import { MemoryRouter } from "react-router-dom"
+import React from "react"
+
+const meta = {
+  component: NavMenu,
+  args: {
+    mobileMenuIsOpen: true,
```

Comment:
I think that would be a good idea.

I'm happy to let that be a follow-up so that this can get merged and unblock other stuff unless ☝️ would be an easy change.

Reply:
Yeah it looks like it's a bit involved due to how intertwined it is with `<LeftNav/>` and `<TopNavMobile/>`, I'll open a ticket to come back to this, but I've got a story for the Left nav I could PR (it's also not _great_ but we'd at least start getting snapshots)

Reply:
> I'm not sold on this stucture though.

Yeah, for now I would just rename this to `Mobile Nav Menu` or `Mobile <NavMenu />` or something like that, or even rename the whole component to `MobileNavMenu` instead of `NavMenu`, and we can figure out how storybook can/should be organized later on, ideally with input from design.

I'm also open to leaving it as-is for now - it's been called `NavMenu` for a while, so we're not making anything _worse_ by leaving it.

Reply:
I considered renaming the component, but I don't think "MobileNavMenu" really describes it very well, so I figured I'd specify in Storybook that we only see it on mobile, but otherwise we'll have to come back to this I think.

Reply:
> https://github.com/mbta/skate/commit/aafd548bdf9ee7e91e3429e62c990f5697d5959e#r138619465

@joshlarson While fullscreen did make the autogenerated doc page worse, it made the story better and more accurate to what it looks like in Skate, so I'd prefer to leave it and address the issues with the autogenerated doc page later. (that would be the position absolute stuff)
https://65047eaabeb50e4310dc692b-jcukjchhum.chromatic.com/?path=/story/skate-mobile-navmenu--story

Reply:
Ahaaaa - I see.

Yeah that's fair